src/components/molecules/LoadingPage/LoadingPage.jsx

```diff
@@ -27,16 +28,12 @@ const quotes = IS_BURN
 export const LoadingPage = () => {
   const [quote, setQuote] = useState("Loading...");
 
-  useEffect(() => {
-    if (!IS_BURN) return;
-    setQuote(quotes[Math.floor(Math.random() * quotes.length)]);
-    const id = setInterval(() => {
+  useInterval(
+    () => {
       setQuote(quotes[Math.floor(Math.random() * quotes.length)]);
-    }, 1000);
-    return () => {
-      clearInterval(id);
-    };
-  }, [setQuote]);
+    },
+    IS_BURN ? 1000 : false
```

Comment:
From code inspection, it appears this won't run `setQuote` in non-burn mode, because `useInterval` will shortcut out. Which means no loading quote will be set. Is that correct?

Reply:
_nods_ basically. If no 'interval time' is provided, there's no point setting an interval (as it's a required param in the underlying command), so we just return early. I think I wrote this as `| undefined` somewhere else.

Reply:
Though it may make sense to abstract the whole 'show a quote' thing into a sub-component. Then conditionally render that whole component based on `IS_BURN`

Reply:
Yes, or to default the text to `Loading...` and only run the `useInterval` based on `IS_BURN`.

Reply:
You can't conditionally render hooks, they have to be after all your branching flows. So you would need to put the 'should this run or should I bail out?' logic inside the hook itself. Which is sort of what i've done here (though I admit it's not the simplest/most discoverable thing.

I think at the very least this warrants another `note:` entry in the JS docs for `src/hooks/useInterval.ts`, and maybe an `@example` entry that demonstrates using it this way.

Reply:
see https://github.com/sparkletown/sparkle/pull/890#discussion_r536485227

Reply:
Sounds good and it's easy to test, just as long as users will see `Loading...` then my concern is solved here.

Reply:
resolved in https://github.com/sparkletown/sparkle/pull/890/commits/2c8050bf8c7a11cd615de82f3f800358a5f41a77

Reply:
```suggestion
    IS_BURN ? 1000 : undefined
```